### ISSUE TITLE
Limits recalculate doesn't seem to work

### ISSUE THREAD

**Issue by OGreenC** [BentoBoxWorld/addon-limits#20]
**Description**
When i attempt to execute the command "/bsbadmin limits calc OGreenC" in the chat, absolutely nothing happens.

**Server Information:**
[Please complete the following information:]
 - Database being used (YAML, JSON, MySQL, MongoDB):
MySQL
Linux Centos
 - Java Version: [e.g. Java 8]
Java 8

 - BentoBox version: [e.g. 1.7.2.21]
Bentobox 1.4.0
- Addons installed? [Do '/bentobox version' and copy/paste from the console]
Biomes 0.4.5
BSkyblock 1.4.2
Level 1.4.0
Limits 0.2.1-SNAPSHOT
WelcomeWarps 1.4.0

**Additional context**

I can see, that this feature was introduced here #10 

The following error appears in the console, shortly after the command is executed:
```25.05 01:24:45 [Server] INFO OGreenC issued server command: /bsbadmin limits calc OGreenC
25.05 01:24:49 [Server] WARN [BentoBox] Task #60624 for BentoBox v1.4.0 generated an exception
25.05 01:24:49 [Server] INFO java.lang.NullPointerException: null
25.05 01:24:49 [Server] INFO at bentobox.addon.limits.commands.LimitsCalc.tidyUp(LimitsCalc.java:138) ~[?:?]
25.05 01:24:49 [Server] INFO at bentobox.addon.limits.commands.LimitsCalc.lambda$new$0(LimitsCalc.java:56) ~[?:?]
25.05 01:24:49 [Server] INFO at org.bukkit.craftbukkit.v1_13_R2.scheduler.CraftTask.run(CraftTask.java:84) ~[patched_1.13.2.jar:git-Paper-625]
25.05 01:24:49 [Server] INFO at org.bukkit.craftbukkit.v1_13_R2.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:449) ~[patched_1.13.2.jar:git-Paper-625]
25.05 01:24:49 [Server] INFO at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:1010) ~[patched_1.13.2.jar:git-Paper-625]
25.05 01:24:49 [Server] INFO at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:439) ~[patched_1.13.2.jar:git-Paper-625]
25.05 01:24:49 [Server] INFO at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:940) ~[patched_1.13.2.jar:git-Paper-625]
25.05 01:24:49 [Server] INFO at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:837) ~[patched_1.13.2.jar:git-Paper-625]
25.05 01:24:49 [Server] INFO at java.lang.Thread.run(Thread.java:745) [?:1.8.0_77]```

**Comment by tastybento** [comment]
@OGreenC This commit should fix it. The issue appears to be with islands that have absolutely no history of block counting.

**Comment by OGreenC** [comment]
Sounds about right. 
I just installed the addon to my server yesterday.

I'll install the newest version of the addon to our server tonight, and get back to you ASAP.

**Comment by OGreenC** [comment]
Works like a charm!

Thanks!

**Comment by OGreenC** [comment]
Hmm - looks like I celebrated a bit too early.
I couldn't really do much testing yesterday, as it was in the middle of the day, thus loads of online players, who would've demanded my head on a spike, if I had restarted the server to do testing.

The recalculate command works just fine. 
However, I've discovered, that blocks is not counted towards the limit, when placed/broken by the player, and the player is not prohibited to place new blocks, when above said limit.
The only way to update the limits, is by running the /bsb limits calc <name> command, which, as expected, will result in the menu /island limits updating.
